config/log4j2.properties

```diff
@@ -0,0 +1,158 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Should we remove log4j.properties?

Reply:
Yes because otherwise we default to the log4j file so you get the warning everytime you run a command:
```
DEPRECATED: Using Log4j 1.x configuration file $KAFKA_HOME/config/log4j.properties
```